root/utility/formatUserDate.js

```diff
@@ -1,17 +1,24 @@
-// This file is part of MusicBrainz, the open internet music database.
-// Copyright (C) 2015 MetaBrainz Foundation
-// Licensed under the GPL version 2, or (at your option) any later version:
-// http://www.gnu.org/licenses/gpl-2.0.txt
+/*
+ * @flow
+ * This file is part of MusicBrainz, the open internet music database.
+ * Copyright (C) 2015 MetaBrainz Foundation
+ * Licensed under the GPL version 2, or (at your option) any later version:
+ * http://www.gnu.org/licenses/gpl-2.0.txt
+ */
 
 import moment from 'moment';
 import trim from 'lodash/trim';
 
 import 'moment-strftime';
 import 'moment-timezone';
 
-function formatUserDate(user, dateString, options) {
-  let preferences = user ? user.preferences : null;
-  let result = moment(dateString);
+function formatUserDate(
+  user: ?(CatalystUserT | SanitizedEditorT),
+  dateString: string,
+  options?: {[string]: boolean},
```

Comment:
Can we replace the indexer with the specific possible options for a little extra type safety? e.g. `{dateOnly?: boolean}` would catch an error if you accidentally passed `daetOnly`.

Reply:
Done - dateOnly seems to be the only one we have at the moment anyway, so :)